components/demo/demo-page.js

```diff
@@ -86,6 +109,13 @@ class DemoPage extends LitElement {
 			this._noScroll = true;
 		}
 	}
+
+	_handleLanguageChange(e) {
+		const newLanguageCode = e.target[e.target.selectedIndex].value;
+		document.documentElement.dir = newLanguageCode === 'ar' ? 'rtl' : 'ltr';
```

Comment:
If we like this approach, I'll make a separate change to the Intl library so that when you set the language to Arabic, it automatically sets the document's direction.

Reply:
Actually on second thought, the Intl library doesn't set the `lang` on the `<html>` element so I don't think this makes sense. It just listens for mutations to those attributes and adapts accordingly.